drivers/display/Kconfig.rm68200

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2022, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config RM68200
+	bool "RM68200 display driver"
```

Comment:
RM68200 is a MIPI DSI panel driver, that should be mentioned in the commit message as well.

Reply:
Done, I have updated the commit message